src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -2108,27 +2111,6 @@ internal override bool IsReadOnly
             }
         }
 
-        //PROTOTYPE(span): Span and ReadonlySpan should have spanLike marker.
-        //                 For now assume that any "System.Span" and "System.ReadOnlySpan" structs 
-        //                 are span-like
-        private bool IsWellknownSpans()
-        {
-            var originalDef = this.OriginalDefinition;
-
-            if (originalDef.Name != "Span" && originalDef.Name != "ReadonlySpan")
-            {
-                return false;
-            }
-
-            var ns = originalDef.ContainingSymbol as NamespaceSymbol;
-            if (ns?.Name != "System")
-            {
-                return false;
-            }
-
-            return ns.IsGlobalNamespace;
```

Comment:
Our spans were made ref structs when ref structs were introduced (as spans will be eventually). As a result, when detection was later broken we did not notice. 
That is why I am also adding a test where spans are not ref structs.